### PR TITLE
Setting up petfinder api

### PR DESCRIPTION
**Pull request by Galina86**
Set up the Petfinder API access token, and got the API to display information.

Petfinder API docs: https://www.petfinder.com/developers/v2/docs/

### CODE REVIEW

File: src/Components/AnimalCard.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import PetsIcon from "@material-ui/icons/Pets";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  card: {
+    backgroundColor: theme.palette.primary.main,
+  },
+  image: {
+    height: 400,
+    [theme.breakpoints.up("lg")]: {
```
Comment (tdss22) [inline src/Components/AnimalCard.js:20]: 🥇

File: src/Components/AnimalCard.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+      height: 250,
+    },
+  },
+}));
+
+const AnimalCard = (props) => {
+  const { result } = props;
+  const classes = useStyles();
+  const [imageLoading, setImageLoading] = useState(true);
+  let ImageComponent;
```
[the remark this answers is not in the document]
Comment (Galina86) [reply src/Components/AnimalCard.js:30]: I was going to make this more refined in the upcoming PR when I style all these components - the main point for this PR was getting the API to work and display all the information

File: src/Components/AnimalCard.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+  if (result.photos && result.photos[0] && result.photos[0].full) {
```
[the remark this answers is not in the document]
Comment (Galina86) [reply src/Components/AnimalCard.js:32]: same as below :) 
in the next PR - i want to try to display only the animals that have photos

File: src/api/oauth-token.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+const petFinderKey = "c8CiKEI1WZbFyRSzWVhIoAlQUsYdUGaDAsJBrWMlW2LTD0SYGx";
+const petFinderSecret = "vOw4xA1qfoRqeIOeQjB8U9oRHwfy3PIk5WeVQ5bT";
```
[the remark this answers is not in the document]
Comment (Galina86) [reply src/api/oauth-token.js]: done!

**Pull request merged**